clients/frontend-authorization/src/oidc/browser/BrowserAuthorizationClient.ts

```diff
@@ -41,6 +41,16 @@ export interface BrowserAuthorizationClientConfiguration {
   readonly noSilentSignInOnAppStartup?: boolean;
 }
 
+/**
+ * Interface describing per-request configuration options for authorization requests
+ * see: https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest
+ * @public
+ */
+export interface BrowserAuthorizationClientRequestOptions {
+  /** The required action demanded of the user before the authentication request can succeed */
+  prompt?: "none" | "login" | "consent" | "select_account" | string;
```

Comment:
is this mirroring the allowable options that oidc-client-js provides?  I see it's available directly on `userManager.settings.prompt`.

Reply:
This does not cover every option oidc-client-js allows on its sigin functions, but "prompt" is one of them.

Specifying prompt in particular is useful on these functions in the case of 401 errors because a new login may be required, but we don't want to always demand that the user always be prompted afterwards (which is what happens if you specify prompt in the main configuration), as this prevents silent logins and renewals.

Reply:
Okay, I was specifically wondering if oidc-client-js defined something for prompt but if they're using `any` I guess not. Looks good to me then.